<!DOCTYPE html>
<html>
<head>
<meta charset="UTF-8">
<style type="text/css">
h1{
 text-align:center;
 font-size:26pt
 }
h2{
text-align:left;
font-size:22pt
}
h3{
font-size:18pt
}
p{
font-size:15pt
}
 </style>
<title>Shangjie Zou Personal Page</title>
</head>
<body>
<h1>
Shangjie Zou
</h1>

<p style="text-align:center">
<img src="image/photo.png" alt="photo"><br/>
E-mail Address: shangjie_zou@foxmail.com<br/>
Phone: (+86)18819259935
</p>
<h2 style = "color:blue">
EDUCATION
</h2>
<p>
South China Agricultural University(SCAU)&nbsp;&nbsp;&nbsp;&nbsp;Sep 2014– Jun2018<br/>
Bachelor’s Degree in biotechnology (BSc)<br/>
GPA Ranking in Major: 7 among 112 students<br/>
English Proficiency: IELTS band 7.5 (Total)<br/>
</p>
<hr/>

<h2 style = "color:blue">
REPRESENTITIVE　ACADEMIC　PROGRAM　EXPERIENCES
</h2>
<h3>
A novel structure of bidirectional longshort-term memory for disulfide bonding state prediction
</h3>
<p>
Tutors: Dr. Shengxiang Zhang, Prof. Yu Sun.&nbsp;&nbsp;&nbsp;&nbsp;01/2018<br/>
(1)Construct a nested bidirectional long short term memory neural network (GL-BLSTM) by Keras;<br/>
(2)Utilize the GL-BLSTM model to predict disulfide bonding state of proteins (Dataset was obtained from PDB database, an open source database for protein structural data);<br/>
Article:<br/> 
Junshu Jiang, Shangjie Zou, Yu Sun and Shengxiang Zhang. GL-BLSTM: A novel structure of<br/>
bidirectional long-short term memory for disulfide bonding state prediction<br/>
Posted on arXiv preprint (<a href= "https://arxiv.org/abs/1808.03745 ">arXiv:1808.03745[q-bio.QM]</a>), Under peer-review in BMC Bioinformatics.<br/>
</p>

<h3>
Comparison of SVM and Spectral Embedding in Promoter Biobricks' Categorizing and Clustering 
</h3>
<p>
Tutor: None
(1)Get all the E.coli σ70 promoters' sequences which are standardized as biobricks from <a href="http://parts.igem.org/Promoters/Catalog">"Registry of Standard Biological Parts"</a> by web crawler.<br/>
(2)Calculate edit distances of the sequences and reduce the dimension of edit distances' matrix by Laplacian eigenmap.<br/>
(3)Transform the sequences by one-hot strategy and feed the dataset into support vector machine to train a classificational model.<br/>
Article:<a href="https://arxiv.org/abs/1902.05724">arXiv:1902.05724v2 [q-bio.OT]</a>
</p>

<h3>
Evolve the function of LacY transport protein by phage-assisted continuous evolution (PACE)
</h3>
<p>
Tutors: Prof. Chenli Liu, Dr. Jinming Cui.&nbsp;&nbsp;&nbsp;&nbsp;06/2018<br/>
(1)Construct expression vectors and engineered phage for PACE (Mutagenesis plasmid, selection phage and accessory plasmid);<br/>
(2)Conduct continuous evolutionary experiments to improve LacY protein’s capacity of transporting specific substances;<br/>
(3)Evaluate the metabolic performance of evolved LacY by HPLC (high performance liquid chromatography).<br/>
</p>

<h3>
Use CRISPR-Cas9 to knock down PRR5 regulator in Arabidopsis thaliana
</h3>
<p>
Tutor: Prof. Wei Huang.&nbsp;&nbsp;&nbsp;&nbsp;05/ 2017<br/>
(1)Design sgRNA which specifically targets to PRR5 in Arabidopsis thaliana and construct the vector of CRISPR-Cas9;<br/>
(2)Identify transformants and evaluate the effects of PRR5 by conducting quantitative real time polymerase chain <br/>reaction (qRT-PCR) on genes that are expected to be regulated by PRR5;<br/>
(3)Analyse data by k-means clustering, Student’s T-test and ANOVA.<br/>
</p>

<h3>
Light triggered gene expression system 
</h3>
<p>
Tutor: Prof. Haihong Wang.&nbsp;&nbsp;&nbsp;&nbsp;03/2015<br/>
(1)Design and construct a expression vector which consist of three genes and two promoters, including a transcription <br/>factor, a reporter gene (GFP) and a fusion protein consist of a light sensor and a kinase;<br/>
(2)Transform the expression vector into bacteria and use light to control expression of target gene. Utilize fluorescence <br/>microscopeto detect the expression of reporter gene;<br/>
</p>
<hr/>
<h2 style = "color:blue">
AWARDS & HONOURS
</h2>
<h3>
2016 iGEM (international genetically engineered machine competition) 
</h3>
<p>
Second Runner-up<br/>
Gold Prize<br/>
Best New Application Project<br/>
Best Plant Synthetic Biology<br/>
Best Education & Public Engagement<br/>
</p>
<h3>
2017 iGEM
</h3>
<p>
Gold Prize<br/>
</p>
<h3>
1st National Undergraduate Genetic Application Competition 
</h3>
<p>
Top 50
</p>

<h3>
2017 "Challenge Cup" Guangdong College Students’ Extracurricular Academic Competition of Science and Technological Works
</h3>
<p>
Third Prize
</p>

<h3>
Honorary title
</h3>
<p>
Pacemaker to Merit Students (2016)<br/>
Excellent Graduate of South China Agricultural University (2018)
</p>
<h3>
Scholarships:
</h3>
<p>
The Second Prize Scholarship (2018)<br/>
The Second Prize Scholarship (2017)<br/>
The First Prize Scholarship (2016)<br/>
The Second Prize Scholarship (2015)
</p>
<hr/>
<h2 style = "color:blue">
INTENSHIP　EXPERIENCES
</h2>
<p>
Research Intern (Visiting Student)&nbsp;&nbsp;&nbsp;&nbsp; Research Center of Bioengineering, Guangzhou Institute of Advanced Technology, Chinese Academy of Sciences&nbsp;&nbsp;&nbsp;06/2018-05/2019<br/>
Biology Teacher &nbsp;&nbsp;&nbsp;&nbsp; Zhong Hao Educational Technology Co., Ltd&nbsp;&nbsp;&nbsp;08/2017-01/2018
</p>
<hr/>
<h2 style = "color:blue">
PRACTICAL　SKILLS
</h2>
<p>
Programming languages: Python, R, Java<br/>
Statistical Software: SPSS
</p>

</body>
</html>